manifests/a/ams/SolutionAG/ams/Client/X64/9.0.40588/ams.SolutionAG.ams.Client.X64.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: ams.SolutionAG.ams.Client.X64
+PackageVersion: 9.0.40588
+PackageLocale: en-US
+Publisher: ams.Solution AG
+PackageName: ams.Client (Build 40588 / main)

```

```suggestion
PackageName: ams.Client
```
Version information should be left out of PackageName